components/molecule/notification/src/index.js

```diff
@@ -0,0 +1,178 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import Button from '@schibstedspain/sui-atom-button'
+import IconClose from '@schibstedspain/sui-svgiconset/lib/Close'
+import IconSuccess from '@schibstedspain/sui-svgiconset/lib/Linecheck'
+import IconError from '@schibstedspain/sui-svgiconset/lib/Lineerror'
+import IconInfo from '@schibstedspain/sui-svgiconset/lib/Lineinfo'
+import IconWarning from '@schibstedspain/sui-svgiconset/lib/Linewarning'
+import cx from 'classnames'
+
+const baseClass = 'sui-MoleculeNotification'
+
+const ICONS = {
```

Comment:
You shouldn't be using fixed icons as this is not scalable to all brands, we need to be able to use different icons than this onces.